autoinstrumentation/nodejs/Dockerfile

```
@@ -4,17 +4,16 @@
 #   update the `NODE_OPTIONS` environment variable accordingly.
 #   In the first stage, install all the required packages in one custom directory.
 #   Then in the second stage, copy the directory to `/instrumentation`.
-# - Ensure you have `newrelic`, `@newrelic/aws-sdk`, `@newrelic/koa`, `@newrelic/superagent`, and `@newrelic/native-metrics`
+# - Ensure you have `newrelic`, which also contains instrumentation for `aws-sdk`, `koa`, `superagent`, and `native-metrics`.
 # - Grant the necessary access to `/instrumentation` directory. `chmod -R go+r /instrumentation`
-FROM node:20.12 AS build
+ARG NODE_RUNTIME_VERSION=20.12
```

Shouldn't this be:
```suggestion
ARG NODE_RUNTIME_VERSION=20
```

The list of images is at https://hub.docker.com/_/node/

Specifying a very specific release version like `20.12` is going to introduce a lot of churn in this file/repo.

---

I also prefer using the broader caregory of runtime version, though I wasn't sure if there was a particular requirement driving the choice of this specific one.